### PR TITLE
added api/posts and api/users routes

### PR DESCRIPTION
**Pull request by hkapk**

### CODE REVIEW

File: controllers/api/comment-routes.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            'created_at',
+        ],
+        include: [
+            {
+                model: Comment,
+                attributes:[`'id',
+                'comment_text', 
+                'post_id', 
+                'user_id', 
+                'post_id'`],
```
Comment (kngurley25) [inline controllers/api/comment-routes.js]: post_id is listed twice here

File: controllers/api/post-routes.js [hunk 2]
```diff
@@ -69,12 +64,12 @@ router.get('/:id', (req, res) => {
     });
 });
 
-router.post('/',   (req, res) => {
-  // expects {title: 'Taskmaster goes public!', post_url: 'https://taskmaster.com/press', user_id: 1}
+router.post('/', (req, res) => {
+  
   Post.create({
     title: req.body.title,
     post_content: req.body.post_content,
```
Comment (kngurley25) [inline controllers/api/post-routes.js]: I think this should match Post model:
user_id
title
description
start_date
end_date

File: controllers/api/post-routes.js [hunk 3]
```diff
@@ -84,7 +79,7 @@ router.post('/',   (req, res) => {
 });
 
 
-router.put('/:id', withAuth,  (req, res) => {
+router.put('/:id', (req, res) => {
   Post.update(
     {
       title: req.body.title
```
Comment (kngurley25) [inline controllers/api/post-routes.js]: I think we should add to edit all fields

Comment (hkapk) [reply controllers/api/post-routes.js]: I think this is fixed now

**Review by kngurley25 — APPROVED** [review]

**Pull request merged**